### PR TITLE
Directives now Working

### PR DESCRIPTION
**Pull request by SalamanderMan**
Resolved issue in the game square test, implemented player selection test

### CODE REVIEW

**Comment by SalamanderMan** [comment]
Should say Directive tests now working..

File: testing/game-model-test.js [hunk 1]
```diff
@@ -82,7 +82,7 @@
             expect(gameModel.typePlayer2).to.equal('human');
         });
 
-
+        it('Check ')
```
Comment (keithbarrow) [inline testing/game-model-test.js]: ?

File: testing/directives-test.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
    'use strict';
-    describe('Game square tests', function(){
+    describe('Directive tests', function(){
        var compile;
        var rootScope;
+       var gameSquare;
+       var squareNumber;
 
-        beforeEach(module('tombola.noughtsAndCrossesApp.directive'));
+        /*beforeEach(module('tombola.noughtsAndCrossesApp.directive'));*/
```
Comment (keithbarrow) [inline testing/directives-test.js]: Don't commit commented code, uncomment or remove. As my old boss used to say "Shit or get off the pot"

File: testing/audio-service-test.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/**
+ * Created by andrew.welsh on 14/04/2015.
```
Comment (keithbarrow) [inline testing/audio-service-test.js:2]: Shit or get off the pot.

File: testing/directives-test.js [hunk 4]
```diff
@@ -1,15 +1,14 @@
-/*
 (function(){
```
Comment (keithbarrow) [inline testing/directives-test.js:1]: This whole thing is commented out - no the wonder it isn't failing.

**Comment by keithbarrow** [comment]
This isn't even a unit of work, so closing.

**Pull request closed without merging**